torch/testing/_internal/common_methods_invocations.py

```diff
@@ -12400,15 +12228,16 @@ def ref_pairwise_distance(input1, input2):
                     autodiff_nonfusible_nodes=['aten::remainder'],),
     BinaryUfuncInfo('__rpow__',
                     op=torch.Tensor.__rpow__,
-                    dtypes=all_types_and_complex_and(torch.bfloat16, torch.half, torch.bool),
```

Comment:
Is removing `bool` here intentional?

Reply:
Yes! Although this is kinda confusing.

The way OpInfos think about dtype support is that all inputs can be in the specified dtype. So `__rpow__` supporting bool means:

```
t = torch.tensor((True, False))
True ** t
```

should work, but that throws an error:

```
RuntimeError: "pow" not implemented for 'Bool'
```

So it is correct that `__rpow__` does not support the bool datatype. It does supporting type promoting from bool, however, like so:

```
t = torch.tensor((True, False))
2 ** t
: tensor([2, 1])
```

Verifying the latter case works is actually a gap in our current testing because we only spot check a subset of dtypes in test_binary_ufuncs.py's test_type_promotion, and we do not check that unsupported dtypes can be promoted to supported dtypes anywhere, I think.